docs/UserGuide.md

```diff
@@ -63,8 +63,23 @@ than current GUI apps in the industry.
 
 ## **Glossary**
 
+### Definitions
+
 Here are some descriptions of the words we use throughout the User Guide:
-- Parameter: Parameters are like fields in a form you are required to fill up. They include details you would include about the doctor/patient.
+- **Parameter**: Parameters are like fields in a form you are required to fill up.
+They include details you would include about the doctor/patient.
```

Comment:
Feels a bit unnecessary since you already explained these terms in the GUI and either than the patient's list, I don't see the terms being used elsewhere, but of course no harm to just add it in